src/models/connection.rs

```diff
@@ -0,0 +1,96 @@
+use context::GraphQLContext;
+use models::*;
+
+pub struct Edge<T> {
+    pub cursor: String,
+    pub node: T,
+}
+
+// TODO: Make Count Lazy
```

Comment:
Can this be moved to a more relevant place?

Reply:
What do you mean by more relevant? See #13